var/spack/repos/builtin/packages/margo/package.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Margo(AutotoolsPackage):
+    """Argobots bindings to the Mercury RPC API"""
+
+    homepage = "https://xgitlab.cels.anl.gov/sds/margo"
+    url      = "https://xgitlab.cels.anl.gov/sds/margo/-/archive/v0.4.3/margo-v0.4.3.tar.gz"
+    git      = "https://xgitlab.cels.anl.gov/sds/margo.git"
+
+    maintainers = ['fbudin69500', 'chuckatkins']
+
+    version('develop', branch='master')
+    version('0.4.3', sha256='61a634d6983bee2ffa06e1e2da4c541cb8f56ddd9dd9f8e04e8044fb38657475')
+
+    variant('shared', default=True,
+            description='Also build shared libraries')
+
+    depends_on('autoconf',  type='build')
+    depends_on('automake',  type='build')
+    depends_on('libtool',   type='build')
+    depends_on('m4',        type='build')
+    depends_on('pkgconfig', type='build')
+
+    depends_on('mercury')
+    depends_on('argobots')
```

Comment:
Does this need to be `depends_on('argobots@1.0rc1:')`?

Reply:
yes. Corrected.